js-exercises/animal-sort/animalSort.js

```diff
@@ -0,0 +1,24 @@
+const animalSort = animals => {
+  if (!(animals instanceof Array)) {
+    throw Error(`Expected Array as an input, got ${typeof animals}`);
+  }
+
+  if (animals.length === 0) {
+    return [];
+  }
```

Comment:
```suggestion
```

`sort` method returns empty array itself without running callback. no need of this condition explicitly